server/db/migrations/20230302043710-change-plugin-description-limit.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const TABLE = 'Plugins';
+const COLUMN = 'description';
+
+module.exports = {
+	up: (queryInterface, Sequelize) =>
+		queryInterface.changeColumn(TABLE, COLUMN, {
+			type: Sequelize.TEXT,
+			allowNull: true
+		}),
+	down: (queryInterface, Sequelize) =>
+		queryInterface.changeColumn(TABLE, COLUMN, {
+			type: Sequelize.STRING,
+			allowNull: false
```

Comment:
@fetok12 change allowNull in "down" to true. This should be allowed to be null anyways.